### PR TITLE
Data jpa concept implementation

### PR DESCRIPTION
**Pull request by Alexsandr-Yablunovskyi**

### CODE REVIEW

**Review by pastcwdt — APPROVED** [review]
Good, good luck on mentor review.

File: src/main/java/yablunovskyi/bookstore/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import yablunovskyi.bookstore.dto.BookDto;
+import yablunovskyi.bookstore.dto.CreateBookRequestDto;
+import yablunovskyi.bookstore.service.BookService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/api/books")
```
Comment (liliia-ponomarenko) [inline src/main/java/yablunovskyi/bookstore/controller/BookController.java:21]:
```suggestion
@RequestMapping(value = "/books")
```

File: src/main/java/yablunovskyi/bookstore/dto/BookDto.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package yablunovskyi.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```
Comment (liliia-ponomarenko) [inline src/main/java/yablunovskyi/bookstore/dto/BookDto.java:7]: better use class record for all dtos

File: src/main/java/yablunovskyi/bookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+    @Column(nullable = false)
+    private boolean isDeleted = false;
```
Comment (liliia-ponomarenko) [inline src/main/java/yablunovskyi/bookstore/model/Book.java:38]:
```suggestion

    @Column(nullable = false)
    private boolean isDeleted = false;
```

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! Let’s fix minor mistakes before merging;)

**Pull request merged**